### ISSUE TITLE
multiple ino-snackbar instances appear upon changing the value of a story knob

### ISSUE THREAD

**Issue by P1NHE4D** [inovex/elements#81]
### Describe the bug
Storybook generates multiple instances of the `ino-snackbar` component upon changing the value of a single knob.

### To Reproduce
1. Go to https://elements.inovex.de/dist/latest/storybook/?path=/story/notification-ino-snackbar--default-usage
2. Open the footer containing the knobs by pressing `D`
3. Change the value of any knob
4. Click on the corresponding `ino-button` to open the snackbar
5. Multiple instances should appear on the screen even though the button has been pressed only once

### Expected behavior
Only a single instance of the `ino-snackbar` should appear if the button is pressed once.

### Screenshots
<img width="1060" alt="Screenshot 2020-09-28 at 17 32 03" src="https://user-images.githubusercontent.com/44554211/94453099-945e1e00-01b0-11eb-883c-34d05879e541.png">

### Your Environment
 

| Software | Version |
| -------- | ------- |
| @inovex.de/elements | 1.0.2 |
| Browser | Safari |
| Operating System | Mac OS |

**Comment by janivo** [comment]
This type of bug also occurs in the `ino-fab-set` if you change one of the `...-location` attributes, the events on this element trigger `1 + number of knob changes` times.

**Comment by P1NHE4D** [comment]
Apparently, storybook calls the `subscribeToComponentEvents` function every time the value of a knob has changed, which causes the number of event listeners to increase over time.

**Comment by P1NHE4D** [comment]
What is more, storybook does not unsubscribe from the event after switching to another story. Hence, the error even occurs after switching back and forth between different stories.
